File: exercises/beecrowd/1014/1014.go
// https://judge.beecrowd.com/pt/problems/view/1014
package main

import (
	"fmt"
)

func main() {
	var distance int
	var fuel float64

	fmt.Scanln(&distance)
	fmt.Scanln(&fuel)

	fmt.Printf("%.3f km/l\n", float64(distance)/fuel)
}

File: exercises/beecrowd/1014/1014.ts
// https://judge.beecrowd.com/pt/problems/view/1014
import * as fs from 'fs';

const input = fs.readFileSync('/dev/stdin', 'utf8');
const lines = input.split(/\r?\n/);

const distance = Number(lines.shift());
const fuel = Number(lines.shift());

console.log(`${(distance/ fuel).toFixed(3)} km/l`);